packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -23,17 +24,42 @@ import {
   isCompiledStyledTaggedTemplateExpression,
   isCompiledCSSMapCallExpression,
 } from './utils/is-compiled';
+import { isTransformedJsxFunction } from './utils/is-jsx-function';
 import { normalizePropsUsage } from './utils/normalize-props-usage';
 import { visitXcssPropPath } from './xcss-prop';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const packageJson = require('../package.json');
 const JSX_SOURCE_ANNOTATION_REGEX = /\*?\s*@jsxImportSource\s+([^\s]+)/;
-const JSX_ANNOTATION_REGEX = /\*?\s*@jsx\s+([^\s]+)/;
 const DEFAULT_IMPORT_SOURCE = '@compiled/react';
 
 let globalCache: Cache | undefined;
 
+const FindClassicJsxPragmaImport: Visitor<State> = {
```

Comment:
nit:

```suggestion
const findClassicJsxPragmaImport: Visitor<State> = {
```

Reply:
hmm babel plugin handbook sometimes capitalises visitors

https://github.com/jamiebuilds/babel-handbook/blob/master/translations/en/plugin-handbook.md#toc-paths-in-visitors

I guess there's no consistency, even in our repository haha